### DISCUSSION TITLE
Is the current example outdated for v2?

### DISCUSSION THREAD

**Discussion by ncpope** [tailwindlabs/heroicons#788] [Help]
I have tried to import heroicons into my project and am on v2. However, the provided examples have produced an error that I am attempting to import from v1.

Am I doing something wrong or do the examples need to be updated?

### React

```jsx
import { CheckCircleIcon } from '@heroicons/react/24/outline';
import { XMarkIcon } from '@heroicons/react/20/solid';
```

### package.json

```json
"dependencies": {
    "@heroicons/react": "^2.0.10",
},
```

### Error

*Uncaught Error: You're trying to import `@heroicons/react/outline/MenuIcon` from Heroicons v1 but have installed Heroicons v2. Install `@heroicons/react@v1` to resolve this error.*

**Comment by brudolph** [comment]
Check the names of your heroicons imports. In your error it's looking for `MenuIcon` but `MenuIcon` is no longer part of v2. Looks like it was renamed to `Bars3Icon` possibly.

Here is a handy list that shows v1 names and their v2 equivalent. 

https://github.com/tailwindlabs/heroicons/releases/tag/v2.0.0
